modAionImpl/src/org/aion/zero/impl/db/AionRepositoryCache.java

```diff
@@ -12,18 +12,18 @@
 import java.util.Set;
 import java.util.concurrent.locks.ReadWriteLock;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
-import org.aion.interfaces.db.ByteArrayKeyValueStore;
-import org.aion.interfaces.db.ContractDetails;
-import org.aion.interfaces.db.InternalVmType;
-import org.aion.interfaces.db.Repository;
-import org.aion.interfaces.db.RepositoryCache;
+import org.aion.db.impl.ByteArrayKeyValueStore;
 import org.aion.log.AionLoggerFactory;
 import org.aion.log.LogEnum;
 import org.aion.mcf.core.AccountState;
+import org.aion.mcf.db.ContractDetails;
 import org.aion.mcf.db.IBlockStoreBase;
+import org.aion.mcf.db.InternalVmType;
+import org.aion.mcf.db.Repository;
+import org.aion.mcf.db.RepositoryCache;
 import org.aion.precompiled.ContractFactory;
 import org.aion.types.AionAddress;
-import org.aion.vm.api.types.ByteArrayWrapper;
+import org.aion.util.types.ByteArrayWrapper;
```

Comment:
This is also a good candidate for `aion_types`, but this can be left as a later TODO.